packages/dynamic_forms_generator/lib/src/generator/model_generator/element_model_generator.dart

```diff
@@ -15,31 +14,27 @@ abstract class ElementModelGenerator {
   String generatePrelude();
   String generateBody();
 
-  dynamic getDefaultValue(String type) {
+  String getDefaultValue(String type) {
     switch (type) {
       case "bool":
         {
-          return false;
+          return "false";
         }
       case "string":
         {
           return "\"\"";
         }
-      case "decimal":
-        {
-          return Decimal.fromInt(0);
-        }
       case "int":
         {
-          return 0;
+          return "0";
         }
       case "double":
         {
-          return 0.0;
+          return "0";
         }
       case "dateTime":
         {
-          return DateTime.now();
+          return DateTime.now().toString();
```

Comment:
I think this won't work. It will return "2019-09-24 12:17:45.720956" and the generated line: 
`myProperty = 2019-09-24 12:17:45.720956`
won't compile. I think it should be "null" or something like  "DateTime(1900, 1, 1, 0, 0, 0)", but null would probably be better default value.

Reply:
Fixed